components/dateinput/events.md

```diff
@@ -12,139 +12,114 @@ position: 20
 
 This article explains the events available in the Telerik DateInput for Blazor:
 
+* [OnBlur](#onblur)
 * [OnChange](#onchange)
 * [ValueChanged](#valuechanged)
-* [OnBlur](#onblur)
 
-## OnChange
-
-The `OnChange` event represents a user action - confirmation of the current value. It fires when the user presses `Enter` in the input, or when the input loses focus.
+## OnBlur
 
-The date input is a generic component, so you must provide either a `Value`, or a type to the `T` parameter of the component.
+The `OnBlur` event fires when the component loses focus.
 
->caption Handle OnChange
+>caption Handle the OnBlur event
 
 ````CSHTML
+@* You do not have to use OnChange to react to loss of focus *@
+
 @result
-<br />
 
-<TelerikDateInput T="DateTime" OnChange="@MyOnChangeHandler"></TelerikDateInput>
+<TelerikDateInput @bind-Value="@DateInputValue"
+                  OnBlur="@OnBlurHandler">
+</TelerikDateInput>
+
+@code{
+    private string result = string.Empty;
 
-@code {
-    string result;
+    private DateTime? DateInputValue { get; set; } = DateTime.Now;
 
-    private void MyOnChangeHandler(object theUserInput)
+    private void OnBlurHandler()
     {
-        // the handler receives an object that you may need to cast to the type of the component
-        // if you do not provide a Value, you must provide the Type parameter to the component
-        result = string.Format("The user entered: {0:dd/MMM/yyyy}", (DateTime)theUserInput);
+        result = string.Format("BLUR fired, current value is {0:dd/MMM/yyyy}.", DateInputValue);
     }
 }
 ````
 
 @[template](/_contentTemplates/common/general-info.md#event-callback-can-be-async)
 
->tip The `OnChange` event is a custom event and does not interfere with bindings, so you can use it together with models and forms.
+
+## OnChange
+
+The `OnChange` event represents a user action - confirmation of the current value. It fires when the user presses `Enter` in the input, or when the input loses focus.
```

Comment:
Use an em dash instead of a hyphen when adding an explanation. The[ em dash rules](https://docs.telerik.com/style-guide/punctuation#lists) about lists also apply to paragraphs.
```suggestion
The `OnChange` event represents a user action—confirmation of the current value. It fires when the user presses `Enter` in the input or when the input loses focus.
```